src/Chartjs/Resources/assets/dist/controller.js

```diff
@@ -55,7 +55,7 @@ var _default = /*#__PURE__*/function (_Controller) {
     value: function connect() {
       var payload = JSON.parse(this.element.getAttribute('data-view'));
 
-      if (!payload.options.length) {
+      if (!Array.isArray(payload.options) && 0 === payload.options.length) {
```

Comment:
Hm.. If it is not an array AND length is zero.. 

Doesn't that mean that `payload.options = {}`?

Reply:
In JS, `[] !== {}`, which is the problem here, as `json_encode` creates `[]` when we would like `{}`

Reply:
> In JS, [] !== {},

Yes,

> as json_encode creates [] when we would like {}

But, that is not what I read... 

If `payload.options = []` then `!Array.isArray(payload.options)` will be false and you will never set `payload.options = {}`

Reply:
What we can do is to always have a default value for `options`?

Reply:
Right! I have an extra `!`